app/assets/stylesheets/basic.scss

```diff
@@ -1,19 +1,15 @@
 
 @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+@import url(https://fonts.googleapis.com/css?family=Muli:400,300italic,300);
+@import url(https://fonts.googleapis.com/css?family=Alegreya+Sans+SC:800);
 
  /************** BASIC STYLING *****************************/
 
-#content {
-  background-color: #FFF;
-  margin-bottom: 6rem;
-  margin-top: 6rem;
-}
-
 #background {
   display: flex;
   flex-flow: column;
   height: 100%;
-  background: url("http://freephotos.atguru.in/hdphotos/forest-landscape-wallpaper/forest-sun-wallpaper-landscape-nature-wallpaper-2560-1600-widescreen-1080.jpg") no-repeat center center fixed;
+  background: url("https://visualmodo.com/wp-content/uploads/2014/02/02_Bokeh-Background.jpg") no-repeat center center fixed;
```

Comment:
URL `https://visualmodo.com/wp-content/uploads/2014/02/02_Bokeh-Background.jpg` should not contain protocol or domain